### ISSUE TITLE
Fix e2e

### ISSUE THREAD

**Issue by seanknox** [Azure/kashti#161]
Currently fails due to what appears to be changed HTML/CSS:

```
1) kashti App sidebar should have a link to the Kashti docs
  - Failed: No element found using locator: By(css selector, *[id="kashti-docs"])

2) kashti App sidebar should have a link to the Kashti issues
  - Failed: No element found using locator: By(css selector, *[id="kashti-issues"])
```

**Comment by flynnduism** [comment]
My bad, I'll correct these.

**Comment by seanknox** [comment]
Oh no need! These are fixed. Closed via #162
